webpack/assets/javascripts/react_app/constants.js

```diff
@@ -10,18 +10,23 @@ export const getControllerSearchProps = (
   controller,
   id = `searchBar-${controller}`,
   canCreate = true
-) => ({
-  controller,
-  autocomplete: {
-    id,
-    searchQuery: '',
-    url: `${controller}/auto_complete_search`,
-    useKeyShortcuts: true,
-  },
-  bookmarks: {
-    id,
-    url: '/api/bookmarks',
-    canCreate,
-    documentationUrl: getManualURL('4.1.5Searching'),
-  },
-});
+) => {
+  const searchUrl = controller.includes("auto_complete_search") ?
```

Comment:
I really dislike APIs that try to be smart. This does fuzzy matching and that's a bad pattern. Please don't do this. If anything, create a new parameter. It's best if that contains the query string as a hash/dict/map (not sure how JS calls it) so it can be properly escaped.

Reply:
Added it as a new parameter `apiParams`.